.circleci/config.yml

```diff
@@ -66,18 +63,8 @@ jobs:
           name: Update pip
           command: pip install -U pip
       - run:
-          name: Bypass numpy issue
-          command: pip install numpy==1.15.4
-      - run:
-          name: Install dependencies
-          command: pip install --progress-bar off -e .[dev]
-      - run:
-          name: Run tests
-          command: PYTHONHASHSEED=42 pytest -m "not figure" -n 4
-
-    environment:
-          MPLBACKEND: agg
-          COLUMNS: 180
+          name: Install SunPy
+          command: pip install --progress-bar off -e .[all,dev]
```

Comment:
I think dev is a superset of all btw.

Reply:
WELL WHY NOT MAKE SURE THAT BOTH WORK

Reply:
In fact i should puit inall the key words to make sure they work